api/reference/conditions.adoc

```diff
@@ -0,0 +1,156 @@
+
+[[conditions]]
+== Conditions
+
+Management of Conditions is provided via the following collection:
+
+[source,data]
+----
+/api/conditions
+----
+
+Full CRUD actions on conditions is available:
```

Comment:
Same question as Tim's above on capitalization - condition is capitalized inconsistently